fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/CommonItemTags.java

```diff
@@ -0,0 +1,82 @@
+
+import net.minecraft.item.Item;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.common.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.ItemTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class CommonItemTags {
+	// Tool tags
+	public static final TagKey<Item> PICKAXES = register("pickaxes");
+	public static final TagKey<Item> SHOVELS = register("shovels");
+	public static final TagKey<Item> HOES = register("hoes");
+	public static final TagKey<Item> AXES = register("axes");
+	public static final TagKey<Item> SHEARS = register("shears");
+	/**
+	 * For throwable weapons, like Minecraft tridents.
+	 */
+	public static final TagKey<Item> SPEARS = register("spears");
+	public static final TagKey<Item> SWORDS = register("swords");
+	public static final TagKey<Item> BOWS = register("bows");
+	// Ores and ingots
+	public static final TagKey<Item> IRON_INGOTS = register("iron_ingots");
+	public static final TagKey<Item> IRON_ORES = register("iron_ores");
+	public static final TagKey<Item> GOLD_ORES = register("gold_ores");
+	public static final TagKey<Item> GOLD_INGOTS = register("gold_ingots");
+	public static final TagKey<Item> REDSTONE_DUSTS = register("redstone_dusts");
+	public static final TagKey<Item> REDSTONE_ORES = register("redstone_ores");
+	public static final TagKey<Item> COPPER_INGOTS = register("copper_ingots");
+	public static final TagKey<Item> COPPER_ORES = register("copper_ores");
+	public static final TagKey<Item> ORES = register("ores");
+	public static final TagKey<Item> NETHERITE_ORES = register("netherite_ores");
+	public static final TagKey<Item> NETHERITE_INGOTS = register("netherite_ingots");
+	public static final TagKey<Item> QUARTZ_ORES = register("quartz_ores");
+	public static final TagKey<Item> QUARTZ = register("quartz");
+	public static final TagKey<Item> LAPIS_ORES = register("lapis_ores");
+	public static final TagKey<Item> LAPIS = register("lapiss");
+	public static final TagKey<Item> DIAMOND_ORES = register("diamond_ores");
+	public static final TagKey<Item> DIAMONDS = register("diamonds");
+	public static final TagKey<Item> EMERALD_ORES = register("emerald_ores");
+	public static final TagKey<Item> EMERALDS = register("emeralds");
+	public static final TagKey<Item> COAL_ORES = register("coal_ores");
+	public static final TagKey<Item> COAL = register("coal");
+	// Consumables
+	public static final TagKey<Item> FOODS = register("foods");
+	public static final TagKey<Item> POTIONS = register("potions");
+	// Buckets
+	public static final TagKey<Item> WATER_BUCKET = register("bucket/water");
+	public static final TagKey<Item> LAVA_BUCKET = register("bucket/lava");
+	public static final TagKey<Item> MILK_BUCKET = register("bucket/milk");
+	public static final TagKey<Item> EMPTY_BUCKET = register("bucket/empty");
+
+	public static final TagKey<Item> GLASS_BLOCKS = register("glass_blocks");
```

Comment:
```suggestion
	public static final TagKey<Item> GLASS_BLOCKS = register("glass");
```
This would reflect the item ids better.

Reply:
IMO, `_blocks` suffix should be kept.

Reply:
`_blocks` is needed to differentiate it from `_panes`

Reply:
Suffix is fine.